dali/pipeline/data/tensor.h

```diff
@@ -131,6 +136,27 @@ class Tensor : public Buffer<Backend> {
     shape_ = shape;
   }
 
+  /**
+   * @brief Tensor is always backed by contiguous buffer
+   */
+  bool IsContiguous() {
+    return true;
+  }
+
+  /**
+   * @brief Tensor is always backed by contiguous buffer
+   *        Cannot be set to noncontiguous
+   */
+  void SetContiguous(bool contiguous) {
+    DALI_ENFORCE(contiguous, "Tensor cannot be made noncontiguous");
+  }
+
+  using Buffer<Backend>::reserve;
+  // For having complete API, Tensor is not a batch
```

Comment:
Nitpick - add an empty line before; otherwise it looks like a comment inside a function.